apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/ListOrders/OpenOrders.tsx

```diff
@@ -201,6 +201,23 @@ export const OpenOrders = () => {
     order: {cancel: cancelOrder},
   } = useSwap()
 
+  const getFee = React.useCallback(
+    async (utxo: string, collateralUtxo: string, bech32Address: string) => {
+      let fee = '0'
+      setIsLoading(true)
+
+      try {
+        fee = await getCancellationOrderFee(wallet, cancelOrder, {orderUtxo: utxo, collateralUtxo, bech32Address})
+      } catch (error) {
+        Alert.alert(strings.generalErrorTitle, strings.generalErrorMessage(error))
+      }
+
+      setIsLoading(false)
+      return fee
+    },
+    [cancelOrder, strings, wallet],
+  )
+
```

Comment:
can we turn it into a react-query and manually refetch on click (enable: false)? also disable cache.

Reply:
build fee as an Amount with PT.